docs/openapi.yaml

```diff
@@ -514,6 +514,47 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
 
+  /customer/resources:
+    get:
+      description: "Search for publications by customer"
+      security:
+        - CognitoUserPool: [
+          "https://api.nva.unit.no/scopes/backend",
+          "https://api.nva.unit.no/scopes/frontend",
+          "aws.cognito.signin.user.admin"
+        ]
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${SearchAuthAwsResourcesFunction.Arn}:live/invocations
+        responses: { }
+        httpMethod: POST
+        type: AWS_PROXY
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PagedPublicationSearchResponse"
```

Comment:
Is it okay to reuse this one from /resources2?

Reply:
If the schema is correct, then yes, re-use is fine